pages/api/bitbucket/callbacks/install.ts

```diff
@@ -0,0 +1,19 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import { publishMessage } from '../../../../utils/pubsub/pubsubClient';
+
+export default async function handler(req: NextApiRequest, res: NextApiResponse) {
+  const topicName = 'rtapish-fromserver';
```

Comment:
Not exactly happy with the `topicName` 😛

Reply:
topic_name?

Reply:
This is besides the point, but variable names in JS & TS should be camelCase.

I was saying that I didn't like the name `rtapish-fromserver`.